content/articles/choosing-wagtail.md

```diff
@@ -0,0 +1,404 @@
+---
+title: "Choosing Wagtail: A Story of Process"
+date: "2018-06-17"
+type: "post"
+author_name:  "Zarina Mustapha"
+author_url: "https://ctl.columbia.edu/about/team/mustapha/"
+lede: "The DevTeam at the CTL continuously research and evaluate existing and
+new software for possible uses in developing educational technologies at
+Columbia University. This post describes the evaluation process that led us to
+choosing Wagtail, an open source CMS written in Python on the Django framework,
+as one of the tools for us to use at the CTL."
+poster: "poster-choosing-wagtail.jpg"
+poster_sourceurl: ""
+poster_source: ""
+topics: 
+- Research and Development
+tags: [wagtail, django, python]
+---
+
+
+On June 14, 2018, I presented at the Wagtail Space US 2018 conference in
+Philadelphia the process the DevTeam went through that led us to choosing
+Wagtail CMS as one of the tools for us to use at the Center for Teaching and
+Learning (CTL). This post is based on that talk. You can also
+[download the slide here](/docs/wagtailspace-20180614.pdf), in PDF format.
+
+We the DevTeam at the CTL continuously research and evaluate existing and new
+software for possible uses in developing educational technologies at Columbia
+University. In building tools and digital environments for classrooms and
+online learning, our approach is grounded in the
+[CTL’s mission](https://ctl.columbia.edu/about/),
+which is to support and promote the _practice_ and _culture_ of _teaching_ and
+_learning_.
+
+Our projects always begin with an educational hypothesis, one that imagines
+ways that software or digital tools can promote learner-centered objectives. We
+help faculty in supporting the tools they chose, and also explore the frontiers
+of educational technologies that engages students towards better scholarship.
+Our research can also lead us to creating novel solutions for these pedagogical
```

Comment:
I think 'create' would be more clear than 'creating'

Reply:
Fixed